db_etl_homepage_graphs/grapher.py

```diff
@@ -56,6 +58,22 @@ def store_data(date: str, metric: str, svg: str, area_type: str = None,
         cli.upload(svg)
 
 
+def store_data_50_plus(date: str, metric: str, svg: str, area_type: str = None,
+               area_code: str = None):
+    kws = dict(
+        container="downloads",
+        content_type="image/svg+xml",
+        cache_control="public, max-age=30, s-maxage=90, must-revalidate",
+        compressed=False
+    )
+
+    if area_code is not None:
+        path = f"homepage/{date}/{metric}/{area_type}/{area_code}_50plus_thumbnail.svg"
+
+    with StorageClient(path=path, **kws) as cli:
```

Comment:
This needs to be indented so that we don't get undefined behaviour.

Reply:
Done